### ISSUE TITLE
The NYS Health Hotline linked phone number is not accessible when TalkBack is running [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#246]
On the "health ResultC" screen in "My Health Log" tab, has a NYS Health Hotline phone number that is announced by TalkBack with the other text in the "Contact a health care provider" paragraph, but it is not reachable.

![health ResultC - Link - NYS Health Hotline](https://user-images.githubusercontent.com/7034772/92039031-5ea94f00-ed42-11ea-8dec-b44bc093dc32.png)

**Remediation Guidance:**

**Comment by floridemai** [comment]
@jonnyjames Can you check in the next build please. Thanks

**Comment by jonnyjames** [comment]
This issue is now fixed. Closing this issue.
